k-distribution/tests/regression-new/context-labels/context-spec.k.out

```diff
@@ -0,0 +1,638 @@
+kore-repl:  Debug (DebugAppliedLabeledRewriteRule):
+    The rule with label [ CONTEXT.foo1-heat ] at location: context.k:8:16-8:52 was applied
+    on configuration:
+      \and{SortGeneratedTopCell{}}(
+          /* term: */
+          /* T Fn D Sfa */
+          Lbl'-LT-'generatedTop'-GT-'{}(
+              /* T Fn D Sfa */
+              Lbl'-LT-'k'-GT-'{}(
+                  /* T Fn D Sfa */
+                  kseq{}(
+                      /* T Fn D Sfa Cli */
+                      /* Inj: */ inj{SortA{}, SortKItem{}}(
+                          /* T Fn D Sfa Cl */
+                          Lblfoo'LParUndsCommUndsRParUnds'CONTEXT'Unds'A'Unds'A'Unds'A{}(
+                              /* T Fn D Sfa Cl */ Lbla'Unds'CONTEXT'Unds'A{}(),
+                              /* T Fn D Sfa Cl */ Lbla'Unds'CONTEXT'Unds'A{}()
+                          )
+                      ),
+                      /* T Fn D Sfa */ Var'Unds'DotVar1:SortK{}
+                  )
+              ),
+              /* T Fn D Sfa */
+              Lbl'-LT-'generatedCounter'-GT-'{}(
+                  /* T Fn D Sfa */ Var'Unds'Gen0:SortInt{}
+              )
+          ),
+      \and{SortGeneratedTopCell{}}(
+          /* predicate: */
+          /* D Sfa */ \top{SortGeneratedTopCell{}}(),
+          /* substitution: */
+          \top{SortGeneratedTopCell{}}()
+      ))
+Context:
+    (InfoReachability) while applying axioms
+kore-repl:  Debug (DebugAppliedLabeledRewriteRule):
+    The rule with label [  ] at location: context.k:15:8-15:14 was applied
```

Comment:
What rule is this? With no label and on line `context.k:15`. The file only has 14 lines in total.

Reply:
Line 15 is `rule a => 0`. I don't know why kore-repl's `DebugAppliedLabeledRewriteRule` tries to display this one.